README.md

```diff
@@ -62,77 +62,82 @@ Here is an overview example:
 
 ```python
 
-    from c8 import C8Client
-    import time
-    import warnings
-    warnings.filterwarnings("ignore")
-    region = "gdn1.macrometa.io"
-    demo_tenant = "mytenant@example.com"
-    demo_fabric = "_system"
-    demo_user = "user@example.com"
-    demo_user_name = "root"
-    demo_collection = "employees"
-    demo_stream = "demostream"
-    collname = "employees"
-    #--------------------------------------------------------------
-    print("Create C8Client Connection...")
-    client = C8Client(protocol='https', host=region, port=443,
-                         email=demo_tenant, password='hidden',
-                         geofabric=demo_fabric)
-
-    #--------------------------------------------------------------
-    print("Create Collection and insert documents")
-
-    if client.has_collection(collname):
-      print("Collection exists")
-    else:
-      client.create_collection(name=collname)
-
-    # List all Collections
-    coll_list = client.get_collections()
-    print(coll_list)
-
-    # Filter collection based on collection models DOC/KV/DYNAMO
-    colls = client.get_collections(collectionModel='DOC')
-    print(colls)
-
-    # Get Collecion Handle and Insert
-    coll = client.get_collection(collname)
-    coll.insert({'firstname': 'John', 'lastname':'Berley', 'email':'john.berley@macrometa.io'})
-
-    # insert document
-    client.insert_document(collection_name=collname, document={'firstname': 'Jean', 'lastname':'Picard',    'email':'jean.picard@macrometa.io'})
-    doc = client.get_document(collname, "John" )
-    print(doc)
-
-    # insert multiple documents
-    docs = [
-        {'firstname': 'James', 'lastname':'Kirk', 'email':'james.kirk@macrometa.io'},
-        {'firstname': 'Han', 'lastname':'Solo', 'email':'han.solo@macrometa.io'},
-        {'firstname': 'Bruce', 'lastname':'Wayne', 'email':'bruce.wayne@macrometa.io'}
-    ]
-
-    client.insert_document(collection_name=collname, document=docs)
-
-    # insert documents from file
-    client.insert_document_from_file(collection_name=collname, csv_filepath="/home/user/test.csv")
-
-    # Add a hash Index
-    client.add_hash_index(collname, fields=['email'], unique=True)
-
-    #--------------------------------------------------------------
-
-    # Query a collection
-    print("query employees collection...")
-    query = 'FOR employee IN employees RETURN employee'
-    cursor = client.execute_query(query)
-    docs = [document for document in cursor]
-    print(docs)
-
-    #--------------------------------------------------------------
-    print("Delete Collection...")
-    # Delete Collection
-    client.delete_collection(name=collname)
+from c8 import C8Client
+import time
+import warnings
+
+warnings.filterwarnings("ignore")
+region = "gdn1.macrometa.io"
```

Comment:
we can make the region as "gdn.paas.macrometa.io" inline with our production url

Reply:
I agree 100%